src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.dsde.rawls.workspace
+
+import akka.actor.{Props, Actor}
+import org.broadinstitute.dsde.rawls.dataaccess.WorkspaceDAO
+import org.broadinstitute.dsde.rawls.model.Workspace
+import org.broadinstitute.dsde.rawls.workspace.WorkspaceService._
+import org.broadinstitute.dsde.rawls.ws.PerRequest
+import spray.http.StatusCodes
+
+/**
+ * Created by dvoet on 4/27/15.
+ */
+object WorkspaceService {
+  case class SaveWorkspace(workspace: Workspace)
+
+  def props(workspaceServiceConstructor: () => WorkspaceService): Props = {
+    Props(workspaceServiceConstructor())
+  }
+
+  def constructor(workspaceDAO: WorkspaceDAO) = () => new WorkspaceService(workspaceDAO)
```

Comment:
maybe constructor is the wrong word. this is a factory function but factory did not seem right either, or does it?